components/Card.js

```diff
@@ -0,0 +1,24 @@
+import { cardInfo } from '../shared/cardInfo';
+
+const CardDetails = (props) => {
+    return (
+        <a href={props.link.href} className="card" target="_blank">
+            <div className="card-header">
+                <img src={props.link.src} alt={props.link.alt} />
+                <h3>{props.link.header} &rarr;</h3>
+            </div>
+            <p>{props.link.text}</p>
+        </a>
+    )
+}
+
+const Card = (props) => {
+    
+    return (
+        <React.Fragment>
+            { cardInfo.map(card => <CardDetails link={card}/>) }
+        </React.Fragment>
+    )
+}
```

Comment:
What do you think about this? I think the `CardDetails` is small enough to inline.

```suggestion
const Cards = (props) => {
        return (
            <React.Fragment>
                {cardInfo.map((card) => (
                    <a href={props.link.href} className="card" target="_blank">
                        <div className="card-header">
                            <img src={props.link.src} alt={props.link.alt} />
                            <h3>{props.link.header} &rarr;</h3>
                        </div>
                        <p>{props.link.text}</p>
                    </a>
                ))}
            </React.Fragment>
        );
    };
```

Reply:
My suggestion also renamed `Card` to `Cards`, so you would also need to update the filename and export if you agree with this suggestion.